examples/animation/double_pendulum_animated.py

```diff
@@ -86,5 +97,4 @@ def animate(i):
 ani = animation.FuncAnimation(fig, animate, np.arange(1, len(y)),
                               interval=25, blit=True, init_func=init)
 
-#ani.save('double_pendulum.mp4', fps=15)
```

Comment:
Can you leave this line?  It is useful to show _how_ to save a movie even if we do not want to do it as part of building the docs.